dash/testing/browser.py

```diff
@@ -244,10 +252,20 @@ def get_webdriver(self, remote):
             )
         )
 
-    @staticmethod
-    def _get_chrome():
-        options = Options()
-        options.add_argument("--no-sandbox")
```

Comment:
Looks like we lost `--no-sandbox`, is it important?

Reply:
no, I removed this part as the options are fully configurable after this PR